scss/civicrm/common/_modals.scss

```diff
@@ -52,7 +53,13 @@
 
     > .ui-button {
       &[type="button"] {
-        right: 20px !important;
+        right: 20px;
+        top: 20px + $ui-icon-margin;
```

Comment:
Can we just add `margin-top: $ui-icon-margin` here? Or is there a bug with margins?

Reply:
this changed to `margin-top: 0` (since jQuery UI adds a default negative margin)